.redhat-ci.yml

```diff
@@ -0,0 +1,15 @@
+host:
+    distro: fedora/24/cloud
```

Comment:
From my perspective our charter in Project Atomic is to containerize - it's *more efficient* than VMs.  The same reason Travis is migrating to Docker.

So we should be defaulting to eating our own dogfood here.